Generate.py

```diff
@@ -127,6 +127,11 @@ def main(args=None, callback=ERmain):
                 player_id += 1
 
     args.multi = max(player_id - 1, args.multi)
+
+    if args.multi == 0:
+        raise Exception("No individual player files found and number of players is 0. "
```

Comment:
I'd probably want this to be a `ValueError` rather than a base exception, but otherwise looks good.

Reply:
Sure, I'm happy with that.

```suggestion
        raise ValueError("No individual player files found and number of players is 0. "
```